applications/incompressible_navier_stokes/navier_stokes_manufactured/application.h

```diff
@@ -0,0 +1,798 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef APPLICATIONS_INCOMPRESSIBLE_NAVIER_STOKES_TEST_CASES_NAVIER_STOKES_MANUFACTURED_H_
+#define APPLICATIONS_INCOMPRESSIBLE_NAVIER_STOKES_TEST_CASES_NAVIER_STOKES_MANUFACTURED_H_
+
+// ExaDG
+#include <exadg/incompressible_navier_stokes/user_interface/enum_types.h>
+
+namespace ExaDG
+{
+namespace IncNS
+{
+/*
+ * Manufactured solution for incompressible flow of a generalized Newtonian fluid in a hypercube.
+ * (Navier-)Stokes equations:
+ * c     ... scaling the convective term with c = 0 or c = 1
```

Comment:
In the implementation you do not use c, but a boolean. So maybe remove c here and wite above that the convective term can be turned off to consider Stokes as usual in ExaDG.